src/booleans.js

```diff
@@ -1,63 +1,123 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  if (a === true && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function either(a, b) {
-  // your code here
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function none(a, b) {
-  // your code here
+  if (a !== true && b !== true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if (a === true && b === false || a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  if (a === b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  if (a > b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  if (a <= b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isOdd(a) {
-  // your code here
+  if (a%2 !== 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isEven(a) {
-  // your code here
+  if (a%2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isSquare(a) {
-  // your code here
+  if (Math.sqrt(a) % 1 === 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function startsWith(char, string) {
-  // your code here
+  if (string[0] === char) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function containsVowels(string) {
-  // your code here
+let vowels = 'aeiouAEIOU';
+
+for (let i = 0; i < string.length; i++) {
+  if (vowels.indexOf(string[i]) !== -1) {
+    return true;
+    }
+  }
+  return false;
 };
 
+//       const containsVowels = string => {   return Boolean(string.match(/[aeiou]/gi));   };    *regex alternative* //
+
+
+
 function isLowerCase(string) {
-  // your code here
+  if (string == string.toLowerCase()) {
+    return true;
+  }
+  return false;
```

Comment:
Can be simplified as above. Also, its always best to use the strict comparison operator `===` where possible:
```suggestion
  return string === string.toLowerCase();
```